src/hooks/use-get-params.ts

```diff
@@ -1,12 +1,15 @@
 import { useSearchParams } from 'react-router-dom';
+import SearchParam from '../types/search-param';
 
 export default function useGetParams() {
   const [searchParams, setSearchParams] = useSearchParams();
 
-  const search = searchParams.get('search');
-  const sortBy = searchParams.get('sortBy') ?? 'release_date';
-  const filter = searchParams.get('filter') ?? 'all';
-  const offset = Number(searchParams.get('offset'));
+  const { Search, SortBy, Filter, Offset } = SearchParam;
```

Comment:
Optional. It's better not to destructure enums, because it's unclear what's going on.